TMessagesProj/build.gradle

```diff
@@ -33,6 +34,8 @@ dependencies {
     implementation 'com.google.code.gson:gson:2.10'
     implementation 'com.google.guava:guava:31.1-android'
 
+    implementation 'com.github.UnifiedPush:android-connector:2.1.1'
```

Comment:
```suggestion
    implementation 'com.github.UnifiedPush:android-connector:2.2.0'
```

Reply:
to do that it's necessary to change the java version used by the project or it won't work (currently JavaVersion.VERSION_1_8 is used). I'll try to change it and see if it still works